scripts/bootloader/generate_zip.py

```diff
@@ -63,8 +63,8 @@ def get_name(text_wrapper):
         special_info[n]['file'] = n
         special_info[n]['modtime'] = int(path.getmtime(p))
         merged = dict()
-        merged.update(special_info[n])
         merged.update(shared_info)
+        merged.update(special_info[n])
```

Comment:
Why is this moved?

Reply:
This change is propagated from https://github.com/nrfconnect/sdk-nrf/pull/5206/commits/9afe7a5f0133da29e2b2e02d9f48ff38c627be35 I don't know.